tensorflow/stream_executor/kernel.h

```diff
@@ -349,6 +349,7 @@ class TypedKernel : public KernelBase {
         port::InlinedVector<uint8, 4>{arg_ptr, arg_ptr + sizeof(arg)}, sizeof(arg)});
   }
 
+#ifndef GOOGLE_CUDA
```

Comment:
this looks wrong -- or rather, there's a bug somewhere else that needs to be fixed, rather than adding this here